test/e2e/framework.go

```diff
@@ -2815,29 +2811,29 @@ func (data *TestData) copyPodFiles(podName string, containerName string, nsName
 }
 
 // copyNodeFiles copies a file from a Node and save it to specified directory
-func (data *TestData) copyNodeFiles(nodeName string, fileName string, covDir string) error {
+func (data *TestData) copyNodeFiles(fileName string, destDir string) error {
 	// getNodeWriter creates the file with name nodeName-suffix. It returns nil if the file
 	// cannot be created. File must be closed by the caller.
-	getNodeWriter := func(nodeName, fileName, suffix string) *os.File {
-		covFile := filepath.Join(covDir, fmt.Sprintf("%s-%s-%s", nodeName, fileName, suffix))
-		f, err := os.Create(covFile)
+	getNodeWriter := func(fileName string) *os.File {
+		destFile := filepath.Join(destDir, fileName)
+		f, err := os.Create(destFile)
 		if err != nil {
-			_ = fmt.Errorf("error when creating coverage file '%s': %v", covFile, err)
+			log.Infof("Error when creating coverage file '%s': %v\n", destFile, err)
 			return nil
 		}
 		return f
 	}
 
-	// dump the file from Antrea Pods to disk.
-	// a filepath-friendly timestamp format.
-	const timeFormat = "Jan02-15-04-05"
-	timeStamp := time.Now().Format(timeFormat)
-	w := getNodeWriter(nodeName, fileName, timeStamp)
+	// dump the file from Nodes to disk.
+	t := strings.Split(fileName, "/")
+	basename := t[len(t)-1]
```

Comment:
could you use a dedicated path utility function for this: https://pkg.go.dev/path#Base (note that you cannot use the one in `path/filepath` because the test code runs on a different system from the Node).